### ISSUE TITLE
Provide PDF of UML on device

### ISSUE THREAD

**Issue by tmfulton** [UMLGenerator/Android-UML-Generator#17]
It may be nice to also allow the PDF to be emailed, tweeted, etc.

**Comment by JakeShor** [comment]
PlantUML and associated plugins allow for this.  Use FileManager.writePlantUML

**Comment by JakeShor** [comment]
86a5dc27555faad3477c63d4400e7f652b1f9362

**Comment by JakeShor** [comment]
This issue is to generate from a xmi or other source of data a UML diagram on the device through an automated action.  Theoretically we'd be able to write our file, and then directly from there read it to generate this UML diagram as a png or some other image type, and then also display it in our application at the end.

**Comment by mformetal** [comment]
Yes, PlantUML does this (provide PDF or image of UML diagram on the device), but why would we use PlantUML? We'd have to write a file with its specific format, convert that to UML, then find some other service to convert UML -> XMI, as the project specifications say. Why not just write to XMI? To that end, this issue should be reopened, because we haven't provided the PDF of our UML diagram generated from a XMI -> UML converter service.
